recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles.inc

```diff
@@ -0,0 +1,19 @@
+require inc/xt_shared_env.inc
```

Comment:
I think this include file can go to meta-xt-images and be re-used by all products and domains

Reply:
Done